CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.x.x - 2021-05-28
```

Comment:
Vamos utilizar o padrão de `Unreleased` sempre que for abrir pr para não dar conflitos, e não adiciona data, muda ai, pls.

Reply:
Entendi, corrigido agora.

Reply:
Entendi, corrigido agora.